### PR TITLE
Migrate to use new config fargate service.

### PR DESCRIPTION
**Pull request by blairboy362**
# Before merge

- [x] Merge and release https://github.com/alphagov/verify-infrastructure-config/pull/335
- [x] Merge and release https://github.com/alphagov/verify-infrastructure/pull/479
- [x] Fix whatever's caused the staging tests to fail

### CODE REVIEW

File: terraform/modules/hub/files/tasks/hub-saml-engine.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
       },
       {
         "Name": "JAVA_OPTS",
-        "Value": "-Dservice.name=config ${jvm_options} -XX:+HeapDumpOnOutOfMemoryError -Dhttp.proxyHost=\"egress-proxy.${domain}\" -Dhttp.proxyPort=\"8080\" -Dhttps.proxyHost=\"egress-proxy.${domain}\" -Dhttps.proxyPort=\"8080\" -Dhttp.nonProxyHosts=\"www.${domain}|config.${domain}|config-fargate.${domain}|policy.${domain}|saml-soap-proxy.${domain}|saml-soap-proxy.${domain}\" -Dnetworkaddress.cache.ttl=5 -Dnetworkaddress.cache.negative.ttl=5"
+        "Value": "-Dservice.name=config ${jvm_options} -XX:+HeapDumpOnOutOfMemoryError -Dhttp.proxyHost=\"egress-proxy.${domain}\" -Dhttp.proxyPort=\"8080\" -Dhttps.proxyHost=\"egress-proxy.${domain}\" -Dhttps.proxyPort=\"8080\" -Dhttp.nonProxyHosts=\"www.${domain}|config.${domain}|config-fargate.${domain}|config-v2-fargate.${domain}|policy.${domain}|saml-soap-proxy.${domain}|saml-soap-proxy.${domain}\" -Dnetworkaddress.cache.ttl=5 -Dnetworkaddress.cache.negative.ttl=5"
```
Comment (blairboy362) [inline terraform/modules/hub/files/tasks/hub-saml-engine.json:87]:
```suggestion
        "Value": "-Dservice.name=saml-engine ${jvm_options} -XX:+HeapDumpOnOutOfMemoryError -Dhttp.proxyHost=\"egress-proxy.${domain}\" -Dhttp.proxyPort=\"8080\" -Dhttps.proxyHost=\"egress-proxy.${domain}\" -Dhttps.proxyPort=\"8080\" -Dhttp.nonProxyHosts=\"www.${domain}|config.${domain}|config-fargate.${domain}|config-v2-fargate.${domain}|policy.${domain}|saml-soap-proxy.${domain}|saml-soap-proxy.${domain}\" -Dnetworkaddress.cache.ttl=5 -Dnetworkaddress.cache.negative.ttl=5"
```

File: terraform/modules/self-service/ecs.tf [hunk 2]
```diff
@@ -18,7 +18,7 @@ locals  {
     sentry_dsn                         = "arn:aws:ssm:eu-west-2:${data.aws_caller_identity.account.account_id}:parameter/${var.deployment}/${local.service}/sentry-dsn"
     hub_environments                   = var.hub_environments
     hub_environments_legacy            = var.hub_environments_legacy
-    hub_config_host                    = "https://config-fargate.${local.hub_deployment}${var.hub_host}:443"
+    hub_config_host                    = "https://config-v2-fargate.${local.hub_deployment}${var.hub_host}:443"
```
Comment (adityapahuja) [inline terraform/modules/self-service/ecs.tf:21]: We will need to update hub_config_host in self-service's site.tf in verify-infrastructure-config to use config-v2-fargate.

Comment (blairboy362) [reply terraform/modules/self-service/ecs.tf:21]: Have I taken care of that in https://github.com/alphagov/verify-infrastructure-config/pull/336 ?

Comment (adityapahuja) [reply terraform/modules/self-service/ecs.tf:21]: Yes.

Comment (blairboy362) [reply terraform/modules/self-service/ecs.tf:21]: :+1: Thank you.

**Pull request merged**